public/app/plugins/datasource/loki/syntax.ts

```diff
@@ -69,9 +70,10 @@ export const PIPE_PARSERS: CompletionItem[] = [
 
 export const PIPE_OPERATORS: CompletionItem[] = [
   {
-    label: 'label_filter',
-    insertText: 'label_filter',
-    documentation: 'Only available in Loki 2.0+.',
+    label: 'unwrap identifier',
+    insertText: 'unwrap identifier',
```

Comment:
sorry I wasn't clear @ivanahuckova identifier means the user will have to choose a label identifier.

e.g: `| unwrap latency`